packages/cli/src/commands/generate/service/__tests__/fixtures/multiWord.test.js

```diff
@@ -1,4 +1,6 @@
+/*
```

Comment:
I'm commenting out the import for now, since the generated test doesn't actually do anything with the import. I figure when we start building out the auto-generated test assertions we can remove the comments from the generation. Alternatively I can remove the comments (although a linter will warn about the unused import)